torch/_ops.py

```diff
@@ -218,8 +222,14 @@ def dispatch(self, dispatch_key, *args, **kwargs):
             # TODO(voz): The idea behind this is that we do not yet support dispatch by key + mode, only key.
             return self.python_key_mode_table[type(curr_mode)](*args, **kwargs)
 
-        assert dispatch_key in self.py_kernels, dispatch_key
-        return self.py_kernels[dispatch_key](*args, **kwargs)
+        final_key = resolve_key(self, dispatch_key)
+
+        assert not isinstance(
+            final_key, str
+        ), f"C++ fallbacks don't work for PyOperator, please reimplement {final_key}"
```

Comment:
I might not be reading this right, but how can `resolve_key` return a str? It seems to only return torch._C.DispatchKey

Reply:
oops, you're right